package.json

```diff
@@ -116,6 +118,8 @@
     "lazy-child": "^0.3.1",
     "lodash.get": "^4.4.2",
     "node-fetch": "^2.7.0",
+    "react-google-recaptcha": "^3.1.0",
+    "react-google-recaptcha-v3": "^1.11.0",
```

Comment:
we should probably remove this carat `^`

Reply:
also shouldn't leaving these packages in just `dependencies` cover both cases? so we could delete it from devDeps